README.md

```diff
@@ -67,6 +67,7 @@ A super awesome collection of regularly used custom hooks as utils for React.
 *   [useSelectableList](https://react-hooks.org/docs/useSelectableList) - Easily select a single value from a list of values. very useful for radio buttons, select inputs  etc.
 *   [useSessionstorage](https://react-hooks.org/docs/useSessionstorage) - Session storage react hook. Easily manage session storage values.
 *   [useSessionstorageState](https://react-hooks.org/docs/useSessionstorageState) - useState but syncs with sessionstorage
+*   [useSet](https://react-hooks.org/docs/useSet) - React state hook that tracks a Set.
```

Comment:
"A React hook that tracks state in the form of a Set data structure."

Reply:
updated